### PR TITLE
DOC-791, part 1: A - D

### PR DESCRIPTION
**Pull request by freakwriter**
Mostly through everything beginning with A & B, but ran into some questions before I can fully finish this. Sharing the PR in case it's helpful as a frame of reference.

Remaining issues I'm unsure what to do with:

1. **AMQP**: I'm ignoring this one in this first pass and moving it into a separate task for more complicated credentials nodes to review at a future date.
2. **Automizy**: I don't think this app actually still exists. You can't sign up for a trial account, app login doesn't work, Help Center docs are dead, Developer docs are dead, last changelog and blog posts are from 2021.
3. **Autopilot**: has rebranded and seems broken; posted in #mission-docs
4. **Box**: Need clarity on the Note for n8n Cloud users situation and what to do with it (and to re-review the other creds nodes I already did with these in it to ensure consistency once I have clarity on what should be happening)
5. **Brandfetch**: I started to update this but didn't commit my changes because these steps don't work; I suspect the API endpoint is actually broken since they're now .com rather than .io. Trying to figure out if I should update it in this pass or just flag that the API and format seems to have changed a bit...
6. **Azure OpenAI**: This is basically updated, but the hyperlinks all have en-us in them because that's where I'm based. Is there a specific country/language combo I should be using to pull these links? I suspect I could use a VPN for this if I need to pick a European country instead.

### CODE REVIEW

File: docs/integrations/builtin/credentials/airtable.md [hunk 1]
```diff
@@ -15,37 +15,41 @@ You can use these credentials to authenticate the following nodes with Airtable.
...
-    - *schema.bases:read* 
-4. Be sure to select the bases this token can **Access** (or all the bases in the workspace).
-5. Copy the token displayed in the dialog.
-6. In the n8n Airtable credentials dialog select the **Access Token** connection.
-7. Paste the token in the **Access Token** field.
-8. Click on the **Save** button to create the credentials.
-
-## Using OAuth 2
+- Personal Access Token (PAT)
+- Oauth2
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/airtable.md:21]:
```suggestion
- OAuth2
```

File: docs/integrations/builtin/credentials/airtable.md [hunk 2]
```diff
@@ -15,37 +15,41 @@ You can use these credentials to authenticate the following nodes with Airtable.
...
+- `data.records:read`
+- `data.records:write`
+- `schema.bases:read`
+
+## Using OAuth2
+
+1. Open your Airtable [Developer Hub](https://airtable.com/create/tokens).
+2. Under the **OAuth integrations** section, select the **Register new OAuth integration** button.
+3. Provide a name for your OAuth integration.
+4. In the n8n Airtable Oauth2 credentials dialog, copy the **OAuth Redirect URL**.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/airtable.md:42]:
```suggestion
4. In the n8n Airtable OAuth2 credentials dialog, copy the **OAuth Redirect URL**.
```

File: docs/integrations/builtin/credentials/airtable.md [hunk 3]
```diff
@@ -15,37 +15,41 @@ You can use these credentials to authenticate the following nodes with Airtable.
...
+5. In Airtable, paste that redirect URL into the **OAuth redirect URL**.
+6. Then select the **Register integration** button in Airtable.
+7. On the following page, copy the **Client ID** from Airtable and paste it into the **Client ID** parameter in the n8n Airtable Oauth2 credentials dialog.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/airtable.md:45]:
```suggestion
7. On the following page, copy the **Client ID** from Airtable and paste it into the **Client ID** parameter in the n8n Airtable OAuth2 credentials dialog.
```

File: docs/integrations/builtin/credentials/airtable.md [hunk 4]
```diff
@@ -15,37 +15,41 @@ You can use these credentials to authenticate the following nodes with Airtable.
...
+8. In Airtable, select the **Generate client secret** button.
+9. Copy the client secret and paste it into the **Client Secret** parameter in the n8n Airtable OAuth2 credentials dialog.
+10. Select the following scopes in Airtable:
+    - `data.records:read`
+    - `data.records:write`
+    - `schema.bases:read`
+11. Select **Save changes** at the bottom of the page in Airtable.
+12. In the n8n Airtable Oauth2 credentials dialog, select the **Connect my account** button.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/airtable.md:53]:
```suggestion
12. In the n8n Airtable OAuth2 credentials dialog, select the **Connect my account** button.
```

File: docs/integrations/builtin/credentials/bitly.md [hunk 5]
```diff
@@ -6,40 +6,34 @@ contentType: integration
...
+Create a [Bitly](https://www.bitly.com/){:target=_blank .external-link} account.
 
-## Using OAuth
+## Supported authentication methods
 
-/// note | Note for n8n Cloud users
-You'll only need to enter the Credentials Name and click on the circle button in the OAuth section to connect your Bitly account to n8n.
-///
+- API Token
+- Oauth2
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/bitly.md:20]:
```suggestion
- OAuth2
```

File: docs/integrations/builtin/credentials/bitly.md [hunk 6]
```diff
@@ -6,40 +6,34 @@ contentType: integration
...
-5. Enter password.
-6. Use the access token in your Bitly node credentials in n8n.
+## Using OAuth2
 
+/// note | Note for n8n Cloud users
+You'll only need to enter the Credentials Name and select the **Connect my account** button in the OAuth credential to connect your Bitly account to n8n.
+///
 
-![Getting Bitly credentials](/_images/integrations/builtin/credentials/bitly/using-access-token.gif)
+Should you need to configure Oauth2 from scratch or need more detail on what's happening in the Oauth web flow, refer to the [Bitly API Authentication documentation](https://dev.bitly.com/docs/getting-started/authentication/){:target=_blank .external-link} for more information.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/bitly.md:38]:
```suggestion
Should you need to configure OAuth2 from scratch or need more detail on what's happening in the Oauth web flow, refer to the [Bitly API Authentication documentation](https://dev.bitly.com/docs/getting-started/authentication/){:target=_blank .external-link} for more information.
```

**Comment by freakwriter** [comment]
@StarfallProjects I think this one is ready for a real final review, hopefully much tidier than before.

Of note:
- I updated the credentials template slightly to make it easier to standardize some of the OAuth stuff.
- I skipped a couple more complicated nodes or those that have been rebranded or had other major changes, and am capturing those separately to be addressed at a future date.

**Pull request closed without merging**